bundle/edu.gemini.wdba.xmlrpc.server/src/main/java/edu/gemini/wdba/tcc/TargetGroupConfig.java

```diff
@@ -23,8 +21,8 @@
     public static final String TYPE_VALUE="targetgroup";
 
     public static TargetGroupConfig createBaseGroup(final TargetEnvironment env) {
-        final ImList<SPTarget> targets = env.getUserTargets().cons(env.getBase());
-        return new TargetGroupConfig(TccNames.BASE, targets, ImOption.apply(env.getBase()));
+        final ImList<SPTarget> targets = env.getUserTargets().cons(env.getArbitraryTargetFromAsterism());
+        return new TargetGroupConfig(TccNames.BASE, targets, ImOption.apply(env.getArbitraryTargetFromAsterism()));
```

Comment:
Might be worth a TODO:ASTERISM label?  I'm not sure what needs to happen here but somehow the TCC is going to need to know about all the targets, I guess.